ajax/index.html

```diff
@@ -251,102 +252,121 @@ <h1 class="center">Asynchronous Technologies: AJAX/JSON</h1>
       </div><!--Requests and Responses-->
 
       <div class="slide normal">
-        <header><h1>1) Creating an XMLHttpRequest</h1></header>
+        <header><h1>Anatomy of an XMLHttpRequest</h1></header>
         <section class="content">
 <pre>
-var request; 
-try {
-  if (typeof ActiveXObject != 'undefined') {
-    request = new ActiveXObject('Microsoft.XMLHTTP');
-  } else if (window["XMLHttpRequest"]) {
-    request = new XMLHttpRequest();
-  }
-} catch (e) {
-  request = false;
-}
-if (!request ) alert("Error initializing XMLHttpRequest!");
+// instantiate a new request
+var request = new XMLHttpRequest(); 
+
+// add event listeners
+request.onload = function() = {
```

Comment:
what do you think about using addEventListener('load')? thats how we usually teach DOM event listeners now. does that work here?